services/worker/src/worker/executor.py

```diff
@@ -198,13 +198,13 @@ def is_worker_alive(self, worker_loop_executor: OutputExecutor) -> bool:
         try:
             worker_loop_executor.stop()  # raises an error if the worker returned unexpected exit code
         except ProcessExitedWithError as err:
-            explanation = f"exit code f{err.exit_code}"
+            explanation = f"exit code {err.exit_code}"
```

Comment:
@lhoestq : ultra nit, but: was this `f` char wanted? or a typo?

Reply:
it's a typo